ansible/roles/vm_set/library/vm_topology.py

```diff
@@ -807,26 +970,28 @@ def main():
             ptf_bp_ip_addr = module.params['ptf_bp_ip_addr']
             ptf_bp_ipv6_addr = module.params['ptf_bp_ipv6_addr']
 
-            if module.params['dut_mgmt_port']:
-                net.bind_mgmt_port(mgmt_bridge, module.params['dut_mgmt_port'])
+            if module.params['duts_mgmt_port']:
```

Comment:
If `external_port` is defined and `dut_mgmt_port` will be an empty string `""`, in this case, `duts_mgmt_port` will be like `["", "", "", ""] if the testbed contains multiple DUTs. `["", ""]` will be evaluated to be `True` in this case.

Reply:
check if string is "" or not.